packages/runtime/src/helpers/compiler.ts

```diff
@@ -31,6 +31,7 @@ const buildMiddlewareFile = async (entryPoints: Array<string>, base: string) =>
       format: 'esm',
       target: 'esnext',
       absWorkingDir: base,
+      external: ['next/dist/compiled/@vercel/og'],
```

Comment:
Without this change the compilation of the middleware would fail as `@vercel/og` contains Node/WASM stuff